pkg/kubernetes/scripts/virtual-machines/utils.jsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @return {Array} names of all enumerable properties
+ */
+
+function getKeys(object) {
+    const result = []
+    for (let propertyName in object) {
+        result.push(propertyName)
+    }
+    return result
+}
```

Comment:
How is that different from `Object.keys(object)`?

Reply:
Removed. It was similar to `Object.keys(object)` but also went down the prototype chain. Verified by experiment that `kubeSelect()` returns results only in own properties. If it's not always true please let me know.